### PR TITLE
Additional Repositories for Ingest

### PR DESCRIPTION
**Pull request by JLLeitschuh**

### CODE REVIEW

File: repos.csv [hunk 1]
```diff
@@ -4646,6 +4650,7 @@ hauner/openapi-generatr-gradle,master,8,,gradlew
 hauner/openapi-processor-gradle,master,8,,gradlew
 haveno-dex/haveno,master,8,,gradlew
 hawtio/hawtio,main,8,,maven
+hawtio/hawtio,master,8,,
```
Comment (JLLeitschuh) [inline repos.csv:4653]:
```suggestion
```

Duplicate

File: repos.csv [hunk 2]
```diff
@@ -1993,6 +1995,7 @@ apache/commons-functor,master,11,,maven
 apache/commons-geometry,master,8,,maven
 apache/commons-graph,master,8,,maven
 apache/commons-imaging,master,8,,maven
+apache/commons-io,master,8,,
```
Comment (JLLeitschuh) [inline repos.csv:1998]: Duplicate
```suggestion
```

File: repos.csv [hunk 3]
```diff
@@ -171,6 +171,7 @@ B3Partners/tailormap,master,8,,maven
 Backblaze/b2-sdk-java,master,8,,gradlew
 Banno/Gordon,main,8,,gradlew
 BaronZ88/MinimalistWeather,master,8,,gradlew
+BaseXdb/basex,master,8,,
```
Comment (JLLeitschuh) [inline repos.csv:174]: Duplicate
```suggestion
```

File: repos.csv [hunk 4]
```diff
@@ -6956,6 +6962,7 @@ kevinYin/ideploy,dev,8,,maven
 kevinpotgieter/gradle-metrics-plugin,master,8,,gradlew
 kevinsawicki/http-request,master,8,,maven
 keyboardsurfer/Crouton,master,8,,gradlew
+keycloak/keycloak,main,8,,
```
Comment (JLLeitschuh) [inline repos.csv:6965]: Duplicate
```suggestion
```

**Comment by traceyyoshima** [comment]
The changes look good, but I'm not sure if the build tool is required.
@tkvangorder can you take a look at this, please?

**Comment by JLLeitschuh** [comment]
According to the README the build tool is optional. Is this not the case?

**Comment by tkvangorder** [comment]
We used to have a utility that would pre-process the CSV, check out just enough files from the repo to determine build type, and then regenerate the CSV with more complete info. But unless we run that thing, the next time the seed job runs, it is basically going to skip those entries that do not have the build tool defined. If we are committing directly to the csv, there is really nothing that is forcing the detection to occur.

**Comment by JLLeitschuh** [comment]
Superseded by #18

**Pull request closed without merging**